src/report/merge.mli

```diff
@@ -0,0 +1,16 @@
+(* This file is part of Bisect_ppx, released under the MIT license. See
+   LICENSE.md for details, or visit
+   https://github.com/aantron/bisect_ppx/blob/master/LICENSE.md. *)
+
+
+
+(** This module merges coverage files into a single coverage file, summing counters pointwise *)
+
+
+val output :
+     to_file:string
+  -> coverage_files:string list
+  -> coverage_paths:string list
+  -> expect:string list
+  -> do_not_expect:string list
+  -> unit
```

Comment:
Could you format this in a way that is consistent with the other reporters?

Bisect_ppx uses a style in which:

1. The presence of `->` near arguments is consistent (`->` by every argument, whether it is first or not &mdash; this allows reordering of arguments to be first without causing a spurious diff on the former first argument's line).
2. The return type is visually distinguished from the arguments by indentation and the lack of `->`.
3. The visual left margin is consistent (no big space on one line followed by `->` on another).

https://github.com/aantron/bisect_ppx/blob/8a10004e3b788ff3f681f552f5c6cdbdc3e8f902/src/report/text.mli#L7-L13